LoRaEngine/LoraKeysManagerFacade/FacadeStartup.cs

```diff
@@ -44,7 +45,7 @@ public override void Configure(IFunctionsHostBuilder builder)
 
 #pragma warning disable CA2000 // Dispose objects before losing scope
             // Object is handled by DI container.
-            _ = builder.Services.AddSingleton(RegistryManager.CreateFromConnectionString(iotHubConnectionString));
+            _ = builder.Services.AddSingleton<IDeviceRegistryManager>(IoTHubRegistryManager.From(RegistryManager.CreateFromConnectionString(iotHubConnectionString)));
```

Comment:
If other suggestions are implemented, CA2000 suppression can be removed and this can be inlined with other builder.Services.Add* calls below on row 56 and so on

Reply:
Tried but CA200 still present on this statement.